src/track/cueinfo.cpp

```diff
@@ -0,0 +1,82 @@
+#include "track/cueinfo.h"
+
+#include "util/assert.h"
+
+namespace {
+const QString kDefaultLabel = ""; // empty string, not null
+}
+
+CueInfo::CueInfo()
+        : m_type(CueInfo::Type::Invalid),
+          m_startPositionMilliseconds(CueInfo::kNoPosition),
+          m_endPositionMilliseconds(CueInfo::kNoPosition),
+          m_hotCueNumber(CueInfo::kNoHotCue),
+          m_label(kDefaultLabel),
+          m_color(Color::kPredefinedColorsSet.noColor) {
+    DEBUG_ASSERT(!m_label.isNull());
+}
+
+CueInfo::CueInfo(
+        CueInfo::Type type,
```

Comment:
"CueInfo::" is redundant in parameter lists of member functions that already belong to the scope of their class

Reply:
Should I remove this? I thought it might improve readability because `Type` without prefix looks overly generic ;-)

Reply:
If we decide to define `CueType` in the outer scope this remark becomes obsolete.